### PR TITLE
Sso db sync

### PR DESCRIPTION
**Pull request by patricksimonian**
Create a playbook for state syncronization

### CODE REVIEW

File: openshift/backup/backup-deploy.json [hunk 1]
```diff
@@ -157,7 +157,7 @@
                     "value": "${BACKUP_PERIOD}"
                   },
                   {
-                    "name": "SSO_PGSQL_MASTER_USER",
+                    "name": "SSO_PGSQL_MASTER_SBOX_148_USER",
```
Comment (patricksimonian) [inline openshift/backup/backup-deploy.json:160]:
```suggestion
                    "name": "SSO_PGSQL_MASTER_USER",
```

File: openshift/backup/backup-deploy.json [hunk 2]
```diff
@@ -166,7 +166,7 @@
                     }
                   },
                   {
-                    "name": "SSO_PGSQL_MASTER_PASSWORD",
+                    "name": "SSO_PGSQL_MASTER_SBOX_148_PASSWORD",
```
Comment (patricksimonian) [inline openshift/backup/backup-deploy.json:169]:
```suggestion
                    "name": "SSO_PGSQL_MASTER_PASSWORD",
```

File: openshift/backup/backup-deploy.silver.param [hunk 3]
```diff
@@ -0,0 +1,39 @@
+#=========================================================
```
Comment (ShellyXueHan) [inline openshift/backup/backup-deploy.silver.param:1]: we are removing the original backup container and switching to helm chart, as a result those oc templates will be gone once I merge in. let's see if we want to turn this into the helm config file later on or just keep the templates for a bit longer.

**Review by ShellyXueHan — APPROVED** [review]
what a big project!! looking great!

qq: plan to move `state_sync` into `ops` sometime?

**Comment by patricksimonian** [comment]
oh should imove it into ops rn?

**Pull request closed without merging**